tf_files/configs/kube-setup-fence.sh

```diff
@@ -62,12 +68,57 @@ if [[ -d "${WORKSPACE}/${vpc_name}_output" ]]; then # update secrets
     g3kubectl create secret generic fence-json-secret --from-file=fence_credentials.json=./apis_configs/fence_credentials.json
   fi
 
-  if ! g3kubectl get secrets/fence-jwt-keys > /dev/null 2>&1; then
-    g3kubectl create secret generic fence-jwt-keys --from-file=./jwt-keys
+  if ! kubectl get configmaps/projects > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/projects.yaml" ]]; then
+      touch "apis_configs/projects.yaml"
+    fi
+    kubectl create configmap projects --from-file=apis_configs/projects.yaml
+  fi
+
+  if ! kubectl get configmaps/user-dir > /dev/null 2>&1; then
+    mkdir -p ./apis_configs/user-dir
+    kubectl create configmap user-dir --from-file=./apis_configs/user-dir
+  fi
+
+  if ! kubectl get secrets/fence-jwt-keys > /dev/null 2>&1; then
+    kubectl create secret generic fence-jwt-keys --from-file=./jwt-keys
   fi
+  
+  if ! kubectl get configmaps/fence-sshconfig > /dev/null 2>&1; then
+    if [[ ! -f "./apis_configs/.ssh/config" ]]; then
+        echo '''
+        Host squid.internal
+          ServerAliveInterval 120
+          HostName cloud-proxy.internal.io
+          User ubuntu
+          ForwardAgent yes
+
+        Host sftp.planx
+          ServerAliveInterval 120
+          HostName sftp.planx-pla.net
+          User foo
+          ForwardAgent yes
+          IdentityFile ~/.ssh/id_rsa
+          ProxyCommand ssh ubuntu@squid.internal nc %h %p 2> /dev/null
+
+        Host cloud-proxy.internal.io
```

Comment:
Should we have an entry for sftp.dbgap ?

Reply:
In production deployment, we will replace it with sftp.dbgap. Right?

Reply:
We don't want to have to add the dbgap entry by hand.  Is there any reason not to put an entry here?

Reply:
We setup sftp.planx-plan.net so that we don't have to manipulate with real sftp dbgap. I don't think we use sftp.planx for other purpose.
We can edit the file ./apis_configs/.ssh/config to configure with the new sftp server
Anyway we can add that entry there